modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+On the **Target details** page, you can view detailed information about a metric target.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
+
+.Procedure
+
+*To view detailed information about a target in the Administrator perspective*:
+
+. Open the {product-title} web console and navigate to *Observe* -> *Targets*.
```

Comment:
The arrow itself might not be incorrect. The SG simply has it as a `>`.

Reply:
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference